pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -497,79 +312,6 @@ func handleInitMarkedDeletePendingObjects(ctx *verifierContext) {
 	}
 }
 
-// Create the object download directories we own
-func createDownloadDirs(objTypes []string) {
-
-	workingDirTypes := []string{"verifier", "verified"}
-
-	// now create the download dirs
-	for _, objType := range objTypes {
-		for _, dirType := range workingDirTypes {
-			dirName := types.DownloadDirname + "/" + objType + "/" + dirType
-			if _, err := os.Stat(dirName); err != nil {
-				log.Debugf("Create %s", dirName)
-				if err := os.MkdirAll(dirName, 0700); err != nil {
-					log.Fatal(err)
-				}
-			}
-		}
-	}
-}
-
-// clear in-progress object download directories
-func clearInProgressDownloadDirs(objTypes []string) {
```

Comment:
If the device crashed or just were rebooted while a verification was in progress, you have some garbage in the verifier directory. Is volumemgr cleaning that up on boot?

Reply:
Based on what I wrote above, the volumemgr will need to create it and clean it up.

verifier moves from "I am a directory everyone knows about and pulls from the public `types.`" to "I am a directory no one but verifier knows about, and has configured internally". It can and should be configured on startup (which allows us to run verifier in other places), but verifier owns it, lock, stock and barrel.

Reply:
Moved this method back to Verifier.